.github/workflows/e2e_android.yml

```diff
@@ -57,8 +61,91 @@ jobs:
       channel_name: va-mobile-build-alerts
       message: 'Starting E2E Android tests. Please see :thread: for results. This process may take a while.'
 
+  matrix-e2e-android:
```

Comment:
Since almost all of this seems to be shared with  the existing `e2e-android` job, maybe you could refactor it out to avoid duplication (I think this is similar to what Dylan suggested in his comment)

Reply:
Dylan and I had a discussion and because refactoring this would probably need either a new reusable workflow or a composite action Dylan and I discussed holding off and doing any refactoring in ticket 7373 (ticket to combine the two workflows).